Libraries/LinkingIOS/RCTLinkingManager.m

```diff
@@ -42,16 +44,29 @@ - (void)stopObserving
   return @[@"url"];
 }
 
++ (BOOL)application:(UIApplication *)app
+            openURL:(NSURL *)URL
+            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
+{
+  postNotificationWithURL(URL);
+  return YES;
+}
+
 + (BOOL)application:(UIApplication *)application
             openURL:(NSURL *)URL
   sourceApplication:(NSString *)sourceApplication
          annotation:(id)annotation
 {
+  postNotificationWithURL(URL);
+  return YES;
+}
+
+void postNotificationWithURL(NSURL *URL)
```

Comment:
Instead pass `self` in here as and change the signature to `postNotificationWithURL(id sender, NSURL *URL)`. Also mark this method as static.